### PR TITLE
ENH: nipype >= 1.0.0 compat fix, var for heuristics finding, etc 

### PR DESCRIPTION
**Pull request by yarikoptic**
Follows up #129

### CODE REVIEW

File: heudiconv/convert.py [hunk 1]
```diff
@@ -381,7 +382,9 @@ def nipype_convert(item_dicoms, prefix, with_prov, bids, tmpdir):
     convertnode.base_dir = tmpdir
     convertnode.inputs.source_names = item_dicoms
     convertnode.inputs.out_filename = op.basename(op.dirname(prefix))
-    convertnode.inputs.terminal_output = 'allatonce'
+    if nipype.__version__.split('.')[0] == '0':
+        # deprecated since 1.0, might be needed(?) before
+        convertnode.inputs.terminal_output = 'allatonce'
```
Comment (satra) [inline heudiconv/convert.py]:
```
else:
    convertnode.terminal_output = 'allatonce'
```

File: heudiconv/cli/run.py [hunk 2]
```diff
@@ -173,7 +173,8 @@ def get_parser():
                         help='Do not catch exceptions and show exception '
                         'traceback')
     parser.add_argument('--command',
-                        choices=('treat-json', 'ls', 'populate-templates'),
+                        choices=('ls', 'populate-templates',
+                                 'treat-jsons', 'sanitize-jsons'),
```
Comment (satra) [inline heudiconv/cli/run.py]: perhaps in alphabetical order.

File: tests/test_heuristics.py [hunk 3]
```diff
@@ -23,22 +24,25 @@
 # this will fail if not in project's root directory
 def test_smoke_converall(tmpdir):
```
Comment (satra) [inline tests/test_heuristics.py]: spelling `convertall`

Comment (yarikoptic) [reply tests/test_heuristics.py]: THANKS! will fix within #130

**Pull request merged**